Models/models.js

```diff
@@ -30,4 +30,22 @@ exports.selectReviewById = (review_id) => {
         }
         return review.rows[0]
     })
+}
+
+exports.selectCommentsByReviewId = (review_id) => {
+    return db.query(
+        `SELECT * FROM reviews WHERE review_id = $1`, [review_id]
```

Comment:
"comments should be served with the most recent comments first"
What SQL method can you use here to achieve this?